src/Backend/Modules/Blog/Engine/Model.php

```diff
@@ -488,58 +479,62 @@ public static function getCategoryId(string $title, string $language = null): in
         );
     }
 
-    /**
-     * Get all data for a given id
-     *
-     * @param int $id The Id of the comment to fetch?
-     *
-     * @return array
-     */
-    public static function getComment($id): array
+    public static function getComment(int $id): array
     {
-        return (array) BackendModel::getContainer()->get('database')->getRecord(
-            'SELECT i.*, UNIX_TIMESTAMP(i.created_on) AS created_on,
-             p.id AS post_id, p.title AS post_title, m.url AS post_url
-             FROM blog_comments AS i
-             INNER JOIN blog_posts AS p ON i.post_id = p.id AND i.language = p.language
-             INNER JOIN meta AS m ON p.meta_id = m.id
-             WHERE i.id = ? AND p.status = ?
-             LIMIT 1',
-            [(int) $id, 'active']
-        );
+        $repository = BackendModel::get('doctrine.orm.default_entity_manager')
```

Comment:
idem

Reply:
fixed in cd9cef81ef540cf410bc31750966a00591aa06ae